protocol/0090-VAMM-automated_market_maker.md

```diff
@@ -237,10 +237,9 @@ As an AMM does not directly place orders on the book this calculation first need
 
 Once these are retrieved, the price / volume points should be combined with a precomputed array of the probability of trading at each price level to calculate the liquidity supplied on each side of the orderbook as defined in [probability of trading](./0034-PROB-prob_weighted_liquidity_measure.ipynb). Once this is calculated, use this value as the instantaneous liquidity score for fee distribution as defined in [setting fees and rewards](./0042-LIQF-setting_fees_and_rewarding_lps.md).
```

Comment:
Do we have any plan to retire the probability of trading?

Reply:
Unsure, but probably not here :)